### ISSUE TITLE
Audio streaming untested on Mark1

### ISSUE THREAD

**Issue by JamesPoole** [JamesPoole/podcast-skill#3]
Any feedback on this appreciated.

**Comment by penrods** [comment]
I just looked at this.  On a Mark 1 I get this at load:

```
21:59:41.128 - mycroft.skills.core:load_skill:106 - INFO - ATTEMPTING TO LOAD SKILL: podcast-skill with ID -639991458
21:59:41.136 - mycroft.skills.core:load_skill:143 - ERROR - Failed to load skill: podcast-skill
Traceback (most recent call last):
File "/usr/local/lib/python2.7/site-packages/mycroft_core-0.9.19-py2.7.egg/mycroft/skills/core.py", line 115, in load_skill
skill = skill_module.create_skill()
File "/opt/mycroft/skills/podcast-skill/__init__.py", line 142, in create_skill
return PodcastSkill()
File "/opt/mycroft/skills/podcast-skill/__init__.py", line 38, in __init__
self.player = vlc.MediaPlayer(self.episode)
File "/usr/local/lib/python2.7/site-packages/vlc.py", line 2756, in __new__
instance = get_default_instance()
File "/usr/local/lib/python2.7/site-packages/vlc.py", line 224, in get_default_instance
_default_instance = Instance()
File "/usr/local/lib/python2.7/site-packages/vlc.py", line 1551, in __new__
return libvlc_new(len(args), args)
File "/usr/local/lib/python2.7/site-packages/vlc.py", line 3903, in libvlc_new
ctypes.c_void_p, ctypes.c_int, ListPOINTER(ctypes.c_char_p))
File "/usr/local/lib/python2.7/site-packages/vlc.py", line 246, in _Cfunction
raise NameError('no function %r' % (name,))
NameError: no function 'libvlc_new'
```

Jumping to VLC is the issue.  Is it possible to use the Mycroft audio service instead, isolating you from this low-level detail?  Kinda like this:  https://github.com/MycroftAI/skill-npr-news/blob/master/__init__.py#L71

**Comment by JamesPoole** [comment]
Thanks for this.
I used the audio service exactly like that npr skill initially but found it wouldn't play a lot of the URLs that I threw at it in my testing. VLC worked great on the desktop install but if it doesn't load at all on Mark 1 I will revert back.  
I also saw that there was the option to change the backend of the audio service to vlc but I am not sure how to actually implement this.

I will make the changes and let you know.
